core/src/com/ericc/the/game/systems/realtime/RenderSystem.java

```diff
@@ -65,6 +70,21 @@ public void update(float deltaTime) {
         int bottom = clamp(0, (int) bottomRight.y - 1, map.height() - 1);
         int right = clamp(0, (int) bottomRight.x, map.width() - 1);
 
+        fov.updateFOV(top, bottom, left, right);
```

Comment:
[wazne] [kolejna okolicznościowa wystawka] to się nie powinno dziać co klatkę. Wywalić do osobnego systemu logicznego